pantry-finder-api/app/serializers/foodbank_serializer.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+# Serializer to strip away the cruft in the locations table
+class FoodbankSerializer < ActiveModel::Serializer
+  attribute :id
+  attributes :address1, :address2, :city, :state, :zip
+  # TODO: attribute  :phone_public_help, key: :phone
+  attributes :fb_name, :fb_nickname
```

Comment:
The `fb` prefix seems unnecessary here. What do you think about mapping all the fb prefixed keys to something more readable like I do here- https://github.com/midohiofoodbank/freshtrak-public/blob/27f17fcc3122a520af348fd71a5e91c0ace158e9/pantry-finder-api/app/serializers/agency_serializer.rb#L6

Reply:
Yes, let's make it consistent